src/app/styles.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import image from '../../public/HomePage.webp';
+
+export const PageContainer = styled.div`
+  width: 100%;
+  min-height: 100vh;
+`;
+
+export const TitleSection = styled.div`
+  width: 100%;
+  height: 100vh; // Use 100vh to make the div fill the entire viewport height
+  background: linear-gradient(
+      222deg,
+      rgba(40, 120, 194, 0.5) 14.2%,
+      rgba(0, 50, 96, 0.5) 76.27%
+    ),
+    url(${image.src}) no-repeat center / cover;
+  display: flex;
+  flex-direction: column;
+  padding-top: 200px;
+  padding-right: 440px;
+  padding-left: 84px;
+  gap: 30px;
+`;
+
+export const ButtonDiv = styled.div`
+  width: 244px;
+  height: 40px;
+`;
+
+export const StatisticsSection = styled.div`
+  width: 100%;
+  height: 146;
+  background-color: white;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0px 8px 20px rgba(0, 0, 0, 0.2);
+  padding-top: 33px;
+  padding-bottom: 35px;
+`;
+export const StatNumber = styled.div`
+  font-size: 35px;
+  color: ${COLORS.blueMid};
+  font-weight: 500px;
+  margin-bottom: 5px;
+`;
+
+// Label for each stat
+export const StatLabel = styled.div`
+  font-size: 22px;
+  color: ${COLORS.greyDark};
+`;
+export const StatContainer = styled.div`
+  text-align: center;
+  /* padding-left: 374px;
+  padding-right: 374px; */
+`;
+export const HowYouCanHelpContainer = styled.div`
+  background-color: ${COLORS.background};
+  width: 100%;
+  height: 557px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  padding-top: 70px;
+`;
+
+// Title and subtitle
+export const Title = styled.h1`
```

Comment:
This is a one `px` difference from the `H2` header defined in `styles/text.tsx`. Could we go ahead and replace this with `H2` on the home page, and remove `Title` from here?